src/BigCommerceLegacyApi/Api/StoreInformation/StoreInformationApi.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace BigCommerce\ApiV2\Api\StoreInformation;
+
+use BigCommerce\ApiV2\Api\Generic\V2ApiBase;
+use BigCommerce\ApiV2\ResourceModels\StoreInformation\StoreInformation;
+
+class StoreInformationApi extends V2ApiBase
+{
+    private const STORE_INFORMATION_ENDPOINT = 'store';
+    private const SYSTEM_TIMESTAMP_ENDPOINT = 'time';
+
+    /**
+     * Returns metadata about a store.
+     *
+     * @return StoreInformation
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function storeInformation(): StoreInformation
+    {
+        $response = $this->getClient()->getRestClient()->get(
+            self::STORE_INFORMATION_ENDPOINT
+        );
+
+        return new StoreInformation(json_decode($response->getBody()));
```

Comment:
Not needed now, but it might be worth adding the `JSON_THROW_ON_ERROR`  at some point